EnglishFilter/sections/content_blocker.txt

```diff
@@ -1560,6 +1561,8 @@ glav.su#@#.c-bl-message
 @@||haaretz.co.il^$elemhide
 ! https://github.com/AdguardTeam/AdguardFilters/issues/57530
 @@||whatsaero.com^$generichide
+! https://github.com/AdguardTeam/AdguardFilters/issues/74653
+@@||downfile.site^$generichide
```

Comment:
It's ok for content blockers. There are limited options for those platforms.

Reply:
Ok.